lib/teiserver_web/controllers/admin/user_controller.ex

```diff
@@ -208,6 +208,54 @@ defmodule TeiserverWeb.Admin.UserController do
     |> render("new.html")
   end
 
+  @spec create_form(Plug.Conn.t(), map) :: Plug.Conn.t()
+  def create_form(conn, _) do
+    conn
+    |> render("create_form.html")
+  end
+
+  @spec create_post(Plug.Conn.t(), map) :: Plug.Conn.t()
+  def create_post(conn, params \\ %{}) do
+    (params["name"] || "" == "")
+
+    passwordfn = fn -> if is_nil(params["password"]) or String.trim(params["password"]) == "" do "password" else params["password"] end end
+    emailfn = fn -> if is_nil(params["email"]) or String.trim(params["email"]) == "" do UUID.uuid1() else params["email"] end end
```

Comment:
you're going to want to setup the formatting tool, either `mix format` or your IDE.

Reply:
should be done now